libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/CycleMapView.java

```diff
@@ -1,6 +1,5 @@
 package net.cyclestreets.views;
 
-import net.cyclestreets.liveride.PebbleNotifier;

```

Comment:
Can you confirm that Pebble support has not been removed? I presume this is some kind of removal of a dependency that didn't actually need to be in this class.